src/interface.jl

```diff
@@ -79,7 +104,7 @@ function solve(
         @assert !MOI.is_valid(lmo.o, cidx)
     end
 
-    direction = ones(n)
+    direction = rand(n)
```

Comment:
rand like this makes the solve function non-deterministic, hence the one instead